lib/data/content/chapter-1.yml

```diff
@@ -1,30 +1,49 @@
 title: "Executive summary"
 menu_title: "Executive summary"
 subtitle: "Chapter 1"
+intro: ""
 photo_credits: "Wulingyuan Scenic and Historic Interest Area, China &copy; aphotostory / Adobe Stock"
 
 case_studies:
   - 
     card_image: /images/case-studies/lanikai-halo-amanda-phung-unsplash.jpg
     card_source: Lanikai halo © Amanda Phung / Unsplash
-    label: 'Box 1'
-    title: 'Authors and contributors'
-    text: "Lead authors:\n
-          Chapter 1: Elise Belle;\n
-          Chapter 2: Edward Lewis;\n
-          Chapter 3: Diego Juffe-Bignoli;\n
-          Chapter 4: Yichuan Shi;\n
-          Chapter 5: Neil Burgess;\n
-          Chapter 6: Heather Bingham;\n
-          Chapter 7: Elise Belle;\n
-          Chapter 8: Heather Bingham;\n
-          Chapter 9: Nina Bhola;\n
-          Chapter 10: Elise Belle\n
-          This report also benefited from the expertise of the following contributing authors (in alphabetical order for each institution):\n
-          William Carney, Colleen Corrigan, Marine Deguignet, Joe Gosling, Jessica Stewart, Judith Walcott and Sylvia Wicander (UNEP-WCMC);\n
-          Thomas Brooks, Marie Fischborn (Box 17), Matea Osti (Box 8) and Elena Osipova (Box 8) (IUCN);\n
-          Jonathan Baillie, Dan Myers, Enric Sala and Therese Tepe (National Geographic Society);\n
-          Sarat Babu Gidda (Convention on Biological Diversity, Secretariat);\n
-          Stuart Butchart and Andrew Plumptre (Chapter 3) (BirdLife International);\n
-          Phil Franks (Box 9) (International Institute for Environment and Development, IIED);\n and
-          Mike Hoffmann and Nisha Owen (Chapter 3) (Zoological Society of London, ZSL)."
\ No newline at end of file
+    label: ''
+    title: 'Contributors and acknowledgements'
+    text: "<span class='bold'>Editors</span>
+      <p>Heather C. Bingham, Edward Lewis, John Tayleur, Cleo Cunningham, Naomi Kingston, Neil D. Burgess and Neville Ash (UNEP-WCMC); Trevor Sandwith (IUCN); Kathy MacKinnon (IUCN WCPA)</p>
```

Comment:
I think you can omit the <p>, <li> and other tags, the partial takes care of that. If you format the text properly it can tell whether you're trying to render a list (like I've done with chapter 11)